docs/drafts/derive-serviceaccount.md

```diff
@@ -0,0 +1,578 @@
+# Derive minimal ServiceAccount required for ClusterExtension Installation and Management
+
+OLM v1 does not have permission to install extensions on a cluster by default. In order to install a [supported bundle](../refs/supported-extensions.md), OLM must be provided a ServiceAccount configured with the appropriate permissions. For more information, see the [provided ServiceAccount](./provided-serviceaccount.md) documentation.
+
+This document serves as a guide for how to derive the RBAC necessary to install a bundle.
+
+### Required RBAC
```

Comment:
I think we should flesh out this section by dividing it into two sub-sections: one for the service account's cluster role and another for the service account's role.